linode/manager
# Feat: [M3-6925] Add Event story + playground

## Description 📝
This PR is the first one of a series of tasks aimed to refactor our events (in JSX). Since it's tedious to test events by triggering an action in the UI, the story and playground will facilitate the refactor by ensuring we're not degrading the existing event messages

## How to test 🧪

1. Pull code locally and navigate to http://localhost:6006/?path=/docs/features-events--documentation
2. Confirm the rendering of the event messages in the story complies with our existing formatting 
3. Confirm the playground helps with crafting accurate event messages
